README.md

```diff
@@ -2,6 +2,16 @@
 
 Haskell application template used at Freckle.
 
+```
+gh repo create --template freckle/haskell-library-template
+```
+
+Setup your repo/package name
+
+```
+sed -i s/haskell-library-template/my-name/ *.*
```

Comment:
Not sure if `*.*` is going to work well -- isn't that top level files with extensions?

```suggestion
sed -i s/haskell-library-template/my-name/ **/*
```

Note that my suggestion isn't particularly compatible. It assumes you're either using a shell that handles `**` correctly, or you know that you aren't and what to do instead. This usage of `-i` was already Linux-only, so I'm not sure how much we'd care.